xbitinfo/xbitinfo.py

```diff
@@ -123,31 +126,37 @@ def get_bitinformation(ds, dim=None, axis=None, label=None, overwrite=False, **k
         >>> ds = xr.tutorial.load_dataset("air_temperature")
         >>> xb.get_bitinformation(ds, dim="lon")
         <xarray.Dataset>
-        Dimensions:  (bit32: 32)
+        Dimensions:  (bit32: 32, dim: 1)
```

Comment:
do we need to have this as an extra dimension if only one dim? could also squeeze. we need the additional dimension fo more than one dim, but for one dim only I'd prefer to squeeze the dim to a coord